TGS3.json

```diff
@@ -8,10 +8,6 @@
         "beautifulsoup4"
       ]
     },
```

Comment:
TGS code is so bad that this is required
```suggestion
    },
    "synchronize_paths": [],
```

Reply:
This is exactly why i pinged you.

I fucking knew some edgecase like this existed.